File: internal/content/testdata/bench/go/merge-sorted-arrays.go
func merge_sorted_arrays(arr1 []int, arr2 []int) []int {
	result := []int{}
	i, j := 0, 0
	for i < len(arr1) && j < len(arr2) {
		if arr1[i] <= arr2[j] {
			result = append(result, arr1[i])
			i++
		} else {
			result = append(result, arr2[j])
			j++
		}
	}
	result = append(result, arr1[i:]...)
	result = append(result, arr2[j:]...)
	return result
}

File: internal/content/testdata/bench/csharp/merge-sorted-arrays.cs
public class Solution {
    public long[] merge_sorted_arrays(long[] arr1, long[] arr2) {
        var result = new List<long>();
        int i = 0, j = 0;
        while (i < arr1.Length && j < arr2.Length) {
            if (arr1[i] <= arr2[j]) result.Add(arr1[i++]);
            else result.Add(arr2[j++]);
        }
        while (i < arr1.Length) result.Add(arr1[i++]);
        while (j < arr2.Length) result.Add(arr2[j++]);
        return result.ToArray();
    }
}
